### PR TITLE
avoid using stream when using update or digest methods

### PR DESCRIPTION
**Pull request by calvinmetcalf**

### CODE REVIEW

**Comment by dcousens** [comment]
Looks OK, but can you explain the changes a little bit more?

**Comment by calvinmetcalf** [comment]
yes previously .update had just done .write but now it avoids putting it's data through the stream just the native crypto does

File: browser.js [hunk 1]
```diff
@@ -24,38 +24,37 @@ HashNoConstructor.prototype._transform = function (data, _, done) {
...
 inherits(Hash, Transform)
 
 function Hash(hash) {
-  Transform.call(this);
+  Transform.call(this)
   this._hash = hash
 }
-
-Hash.prototype._transform = function (data, _, done) {
+Hash.prototype._transform = function (data, enc, done) {
```
Comment (dcousens) [inline browser.js]: new line was removed?

**Comment by dcousens** [comment]
ACK

**Pull request merged**